src/utils/index.js

```diff
@@ -35,6 +35,7 @@ export const TOO_MANY_FILES_REJECTION = {
   message: 'Too many files'
 }
 
+
```

Comment:
I think you forgot the extra line here, please remove. We can then merge the PR. Thanks.

Reply:
done